dpgv4.py

```diff
@@ -321,8 +321,10 @@ def prepare_video_conversion_command(
         "-f", "data",
         "-map", "0:v:0",
         "-r", f"{framerate:g}",
-        "-sws_flags", "lanczos",
-        "-s", f"{width}x{height}",
+        #"-sws_flags", "lanczos",
+        #"-s", f"{width}x{height}",
+        #Prevent wrong width crash
+        "-vf", "scale=min(256\\,192*a):-1,pad=256:192:abs(ow-iw)/2:abs(oh-ih)/2",
```

Comment:
Please use [f-strings](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals) and replace 256 and 192 with `SCREEN_WIDTH` and `SCREEN_HEIGHT` respectively.

Since the `width` and `heigt` variables aren't going to be used anymore, please remove them along with the `calculate_dimensions` function and its test.

An alternative approach for this could be to keep the `calculate_dimensions` function and expand it to calculate not only width and height, but also vertical and horizontal padding. For example, return a named tuple with `width`, `height`, `horizontal_padding` and `vertical_padding` fields, use `width` and `height` for the `scale` filter and `vertical_padding` and `horizontal_padding` for the `pad` filter. This would have the advantage of being more testable.